src/lib/schemas.ts

```diff
@@ -70,8 +70,11 @@ export const detailStoreSchema = portfolioSchema.extend({
 
 export const investmentSchema = z.object({})
 
-export const loginFormSchema = z.object({
-	email: z.string().email({ message: 'Enter a valid email.' }),
+export const emailFormSchema = z.object({
+	email: z.string().email({ message: 'This doesn’t look like a correct email' }),
```

Comment:
Should be localised string
```suggestion
	email: z.string().email({ message: $_('This doesn’t look like a correct email' )}),
```